src/components/common/EthHashInfo/index.tsx

```diff
@@ -37,6 +42,7 @@ const EthHashInfo = ({
   hasExplorer,
 }: EthHashInfoProps): ReactElement => {
   const [fallbackToIdenticon, setFallbackToIdenticon] = useState(false)
+  const shouldPrefix = ethers.utils.isAddress(address)
```

Comment:
```suggestion
  const isAddress = ethers.utils.isAddress(address)
```

Reply:
I would argue that this is more declarative as we don't want to prefix hashes. What do you think?

Reply:
I think it doesn't reflect our real intention in the logic. Perhaps `canPrefix` ? Once it can only prefix if the string is an address. Your call though